test/Microsoft.ML.Tests/TensorFlowEstimatorTests.cs

```diff
@@ -135,7 +135,8 @@ void TestOldSavingAndLoading()
         [ConditionalFact(typeof(Environment), nameof(Environment.Is64BitProcess))] // x86 output differs from Baseline
         void TestCommandLine()
         {
-            var env = new MLContext();
+            // typeof helps to load the TensorFlowTransformer type.
+            Type type = typeof(TensorFlowTransformer);
```

Comment:
Any idea why `typeof` helps?

Reply:
So when we create an instance of a type in Maml, we check if the type\assembly is present in the loaded modules.
Typeof helps to load the assembly involving this type.